metricbeat/module/elasticsearch/index_recovery/index_recovery.go

```diff
@@ -50,7 +50,7 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	config := struct {
 		ActiveOnly bool `config:"index_recovery.active_only"`
 	}{
-		ActiveOnly: true,
+		ActiveOnly: false,
```

Comment:
I made this change to be consistent with the current default value in XPack Monitoring: 

https://github.com/elastic/elasticsearch/blob/ed0571e16caac689a77362bfe4c45c79dcba4d82/x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/collector/indices/IndexRecoveryCollector.java#L41-L45

However, it means that even for the non-xpack-monitoring case we will collect all index recoveries, not just active ones. WDYT of that?

Reply:
Ignore this comment, I was able to make this flag smarter in 4817f48.